org.swi_prolog.swipl.yml

```diff
@@ -0,0 +1,40 @@
+id: org.swi_prolog.swipl
+runtime: org.kde.Platform
+runtime-version: "6.7"
+sdk: org.kde.Sdk
+command: swipl-win
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=host
```

Comment:
why is host permission necessary?

Reply:
being a programming language a more extended access to the system is typically needed. One example would be external drives.

This permission could be removed or replaced with something else once the flatpak works